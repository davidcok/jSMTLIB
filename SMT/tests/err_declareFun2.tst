; declaring an already declared fun
(set-logic QF_UF)
(declare-fun x () Bool)
(push 1)
(declare-fun x () Bool)
