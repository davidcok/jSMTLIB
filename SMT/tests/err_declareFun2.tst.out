success
success
success
(declare-fun x () Bool)
             ^
(error "Symbol x is already defined")
